### ISSUE TITLE
Problem with `tns livesync ios --watch`  on real device 

### ISSUE THREAD

**Issue by tsonevn** [NativeScript/nativescript-cli#2371]
I was able to reproduce this problem only with `nativescript-ui-samples-angular`, while using `livesync` command. The app has been builded successfully, however it hangs on launch  screen. This problem has been reproduced only on ios real device. I am attaching steps for reproducing:
1. `git clone https://github.com/telerik/nativescript-ui-samples-angular`
2. `tns livesync ios --watch`

Error
```
***** Fatal
JavaScript exception - application has been terminated. *****
Native stack trace:
1   0x1004e5dc0 -[TNSRuntime executeModule:referredBy:]
2   0x1000a5438 main
3   0x1804e28b8 <redacted>
JavaScript stack trace:
1   @file:///app/tns_modules/ui/core/dependency-observable.js:73:28
JavaScript error:
file:///app/tns_modules/ui/core/dependency-observable.js:73:28: JS ERROR Error: Property scrollDirection already registered for type ListViewLayoutBase.
```

**Comment by ginev** [comment]
@tzraikov we will take a look at it. thanks!

**Comment by mourice** [comment]
Link not found. Getting the same error for real device. emulator its working fine

**Comment by simonettoa** [comment]
me too, same error and the link is in 404 error! thanks

**Comment by NickIliev** [comment]
Hey @simonettoa from version 2.5.0 the livesync command is obsolete and you can directly execute
```
tns run ios
```
which now will work as livesync --watch (will re-deploy your application after each change made in your application code)

**Comment by simonettoa** [comment]
Hi @NickIliev, thanks for the tip, I know it and I am using just the command 'tns run ios'! In the emulator is working but on the real device it's crushing. I realised now that the error is a little different from the one above. I have to try in the real device because I am testing the push notifications.

The error that I am facing is this below:

`Found peer TypeScript 2.1.6
***** Fatal JavaScript exception - application has been terminated. *****
Native stack trace:
1   0x1014c1e20 -[TNSRuntime executeModule:referredBy:]
2   0x10003d304 main
3   0x180e805b8 <redacted>
JavaScript stack trace:
1   @file:///app/tns_modules/tns-core-modules/ui/core/dependency-observable.js:73:28
JavaScript error:
file:///app/tns_modules/tns-core-modules/ui/core/dependency-observable.js:73:28: JS ERROR Error: Property showOverNavigation already registered for type RadSideDrawer.
`

I have found in stack overflow a person that have the same error (asked in January but no reply). I also tried to get back to other version with git but the error is the same. The only difference is that I updated nativescritp to the last version (2.5.2).

Can you suggest a link or something else (maybe where I can open a new thread?!?!) thanks.

**Comment by danielgek** [comment]
@simonettoa @NickIliev i get the same thing on real device, however if i run the release build it works just fine(`tns run ios --release`)

**Comment by simonettoa** [comment]
@danielgek thanks, it works. I also discovered that there was an error receiving the notification (in my case), that was the cause of the crashing when the notifications was received; the strange thing is that was not a problem in android emulator/real device. 
If I launch the run command without release the same error above appears.
But with '--release' I cannot see the console.log commands. Is it normal or is something else?

**Comment by danielgek** [comment]
yes it is supposed to, in release logs are ommited

**Comment by simonettoa** [comment]
thanks again! Still I don't get why that error is appearing in the 'normal' way, maybe there is some issues with different libraries installed in my project. I also created a simple test just to try the firebase plugin and it's working without the '--release' part.

**Comment by danielgek** [comment]
@simonettoa yes i'm still trying to track down what is causing this for my side too

**Comment by simonettoa** [comment]
I opened an issue in telerik feedback page. 

https://github.com/telerik/nativescript-ui-feedback/issues/137

Connected with that there is a repo that I created with a simple app that has a RadSideDrawer and it's possible to test it. I found a similar issue (don't know if it's related) here
 
https://github.com/telerik/nativescript-ui-feedback/issues/95

**Comment by simonettoa** [comment]
I opened an issue in telerik feedback page.

telerik/nativescript-ui-feedback#137

Connected with that there is a repo that I created with a simple app that has a RadSideDrawer and it's possible to test it. I found a similar issue (don't know if it's related) here

telerik/nativescript-ui-feedback#95
